### DISCUSSION TITLE
About using <Head> component with NextJS 13 and the warning log.

### DISCUSSION THREAD

**Discussion by state303** [mantinedev/mantine#5034] [Q&A]
Hi everyone, 
I am currently developing a new site with mantine + nextjs 13.
Everytime I build, I get the following warning... which is not critical, but feels a bit incomfortable:

```
Warning: You're using `next/head` inside the `app` directory, please migrate to the Metadata API. See https://nextjs.org/docs/app/building-your-application/upgrading/app-router-migration#step-3-migrating-nexthead for more details.
```

And of course, this happens because of the usage of <Head> component as below...
```typescript
<Head> <ColorSchemeScript defaultColorScheme="dark" nonce="hidden"/> </Head>
```

Of course, I can simply ignore the warning and be happy, but just I don't think it would be so difficult to fix.
Does anyone have a solution for this? Just out of curiousity.. :)

**Comment by rtivital** [comment]
See documentation for app router, it does not use Header component – https://mantine.dev/guides/next/#setup-with-app-router
